oar/core/advisory.py

```diff
@@ -749,3 +751,45 @@ def get_overall_grade(self):
         else:
             raise AdvisoryException(
                 f"error when accessing advisory containers - {self.errata_id}")
+
+    def check_kernel_tag(self):
+        # get advisories url
+        """
+        Get kernel tag from advisory build image.
+
+        Returns:
+            bool: True if kernel tagged with early-kernel-stop-ship.
+        """
+        candidate_builds = self._cs.get_candidate_builds()
+        if candidate_builds:
+            for k, v in candidate_builds.items():
+                build_url = "https://errata.devel.redhat.com/api/v1/erratum/"+v+"/builds"
+                logger.info(k+" build url:"+v+"\n")
+                session = requests.Session()
```

Comment:
suggest to impl func at Advisory level to check kernel tag
use http related func from parent class `Erratum` to interact with errata tool

Reply:
I don't the issue is resolved. my understanding of this logic is, rhcos tag checking is happened at advisory level not manager level. most of the funcs should be implemented at advisory level, advisory manager is used to iterate all the valid advisories and execute their funcs finally collect the result and return the expected result. in this case, tag checking should be applied on `image` advisory only, there is no rhcos build attached in other advisories.

Reply:
also for http data exchange, the parent class of `Advisory` provides some builtin funcs, like https://github.com/red-hat-storage/errata-tool/blob/25489e09945fccb29e5bc6e5de3208bca097d0b0/errata_tool/connector.py#L119